client/electron/go_vpn_tunnel.ts

```diff
@@ -67,8 +67,8 @@ export class GoVpnTunnel implements VpnTunnel {
     private readonly routing: RoutingDaemon,
     readonly transportConfig: TransportConfigJson
   ) {
-    this.tun2socks = new GoTun2socks(transportConfig);
-    this.connectivityChecker = new GoTun2socks(transportConfig);
+    this.tun2socks = new GoTun2socks();
```

Comment:
Do we still need these 2 `GoTun2socks()` entities? I forget why you need them. Do you mind adding a comment for future readers?

Reply:
Added a comment